### ISSUE TITLE
python3 compatibility

### ISSUE THREAD

**Issue by bastianh** [lamby/django-slack#11]
Hi,

have you already thought about how to make django-slack compatible with django 1.9?
looks like django don't like the slack_message function imported at load time.
`django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.`

I know there is still some time until django 1.9 release, but maybe you have thought about it already?

bastian

**Comment by lamby** [comment]
> looks like django don't like the slack_message function imported at load time.

Can you post a full traceback? I can't repoduce this under 1.9 alpha.

**Comment by bastianh** [comment]
hmm.. could not reproduce it using a minimal django project. Looks like the import of the function in __init__.py triggers the loading of something else in my project that is not allowed to load at this point. (some model?). When I uncomment the import in __init__.py and import the function from api.py everything works.
The traceback is not really helpful :(
```
Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/bastianh/Development/corp_manager/.venv/lib/python2.7/site-packages/django/core/management/__init__.py", line 350, in execute_from_command_line
    utility.execute()
  File "/Users/bastianh/Development/corp_manager/.venv/lib/python2.7/site-packages/django/core/management/__init__.py", line 342, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/bastianh/Development/corp_manager/.venv/lib/python2.7/site-packages/django/core/management/__init__.py", line 176, in fetch_command
    commands = get_commands()
  File "/Users/bastianh/Development/corp_manager/.venv/lib/python2.7/site-packages/django/utils/lru_cache.py", line 100, in wrapper
    result = user_function(*args, **kwds)
  File "/Users/bastianh/Development/corp_manager/.venv/lib/python2.7/site-packages/django/core/management/__init__.py", line 71, in get_commands
    for app_config in reversed(list(apps.get_app_configs())):
  File "/Users/bastianh/Development/corp_manager/.venv/lib/python2.7/site-packages/django/apps/registry.py", line 137, in get_app_configs
    self.check_apps_ready()
  File "/Users/bastianh/Development/corp_manager/.venv/lib/python2.7/site-packages/django/apps/registry.py", line 124, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.
```

**Comment by bastianh** [comment]
ok .. my fault. When I added your library that was the first error I got .. and since it's the same error message that I got from other libraries I thought it had to do with django 1.9 

I fixed it by uncommenting the import in __init__.py and then I stumbled over the missing python3 compatiblity in backend.py so I fixed that.
 
The problem is that your way of importing the backend silenty ignores the import error that are caused by the missing python 3 compatiblity in the backends.py file but then later causes django to fail to start.
